addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -3082,7 +3081,9 @@ export class OdooEditor extends EventTarget {
      * @param {Node} node
      */
     _cleanForPaste(node) {
-        if (!this._isWhitelisted(node) || this._isBlacklisted(node)) {
+        if (!this._isWhitelisted(node) || this._isBlacklisted(node) ||
+            node.id && node.id.startsWith('docs-internal-guid') ||
```

Comment:
Could you write a new test using html you got from google docs, please ? That way it will be properly tested and I will also be able to easily see the kind of content that we get. Thanks !

Reply:
I have added the test cases for it.